app/views/provider/admin/dashboards/_apiap_search_bar.html.slim

```diff
@@ -0,0 +1,10 @@
+div id=id
+
+javascript:
+  document.addEventListener('DOMContentLoaded', function () {
+    var apis = #{json collection}
+    var placeholder = #{json placeholder}
+    var displayApis = () => {}
+
+    window.ApiFilter({ apis, placeholder, displayApis }, #{json id})
+  })
```

Comment:
You can add to you vscode configs:

```
"files.insertFinalNewline": true
```

and avoid `TrailingBlankLines` cop

Reply:
boom! thanks